Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,35 @@
-﻿namespace Logger;
 
-public static class BaseLoggerMixins
+using System;
+using System.Runtime.CompilerServices;
+using static Logger.BaseLogger;
+
+namespace Logger
 {
-    
+    public static class BaseLoggerMixins
+    {
+        public static void Error(this BaseLogger log, string message, params object[] args)
```

Comment:
```suggestion
        public static void Error(this BaseLogger logger, string message, params object[] args)
```
This should probably be called `logger`, not `log` as it's not a log message

Reply:
Agree